clang-tools-extra/clang-doc/tool/ClangDocMain.cpp

```diff
@@ -131,12 +137,55 @@ std::string GetExecutablePath(const char *Argv0, void *MainAddr) {
   return llvm::sys::fs::getMainExecutable(Argv0, MainAddr);
 }
 
+void GetAssetFiles(clang::doc::ClangDocContext &CDCtx) {
+  std::error_code Code;
```

Comment:
nit: I'd use `Err`. 

There isn't a completely consistent practice in Clang-Doc code, but `FileErr` seems most common. https://llvm.org/docs/ProgrammersManual.html#interoperability-with-std-error-code-and-erroror has a little to say, and the `Error handling` section on that page may be useful, but we don't have a firm convention.

Reply:
```suggestion
  std::error_code Err;
```
```suggestion
  std::error_code FileErr;
```

Reply:
This still uses `Code`. please use a naming convention consistent w/ this tool and other parts of LLVM before marking it resolved.